pkg/syncer/syncer.go

```diff
@@ -91,8 +96,9 @@ func New(from, to *rest.Config, upsertFn UpsertFunc, deleteFn DeleteFunc, handle
 
 		stopCh: stopCh,
 
-		upsertFn: upsertFn,
-		deleteFn: deleteFn,
+		upsertFn:  upsertFn,
+		deleteFn:  deleteFn,
+		namespace: os.Getenv(SYNCER_NAMESPACE_KEY),
```

Comment:
How should syncer react when this env var is unset? Should it panic, or fallback to some default?

Reply:
that should be I thought covered in `inSyncerNamespace`